pkgs/by-name/pl/plus-jakarta-sans/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "plus-jakarta-sans";
+  version = "2.7.1";
+
+  src = fetchzip {
+    url = "https://github.com/tokotype/PlusJakartaSans/releases/download/${version}/PlusJakartaSans-${version}.zip";
+    stripRoot = false;
+    hash = "sha256-+ghkQ4/wxou3FB68ceCwBeGrSCaHJhb16MybvN6mCSc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    cp PlusJakartaSans-$version/*/PlusJakartaSans*.ttf $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.tokotype.com/custom-fonts/plusjakarta";
+    description = "Typeface designed for Jakarta 'City of collaboration' program in 2020";
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ gavink97 ];
+  };
```

`with lib` was used and abused a lot in the past, now set-wide `with`s are frowned upon. `with` with single lists, like `meta.maintainers`, are fine
```suggestion
  meta = {
    homepage = "https://www.tokotype.com/custom-fonts/plusjakarta";
    description = "Typeface designed for Jakarta 'City of collaboration' program in 2020";
    license = lib.licenses.ofl;
    platforms = lib.platforms.all;
    maintainers = with lib.maintainers; [ gavink97 ];
  };
```